bugtool/cmd/configuration.go

```diff
@@ -73,6 +73,11 @@ func defaultCommands(confDir string, cmdDir string, k8sPods []string) []string {
 		"ip rule",
 		"ip -4 route show table 2005",
 		"ip -6 route show table 2005",
+		"ip -4 route show table 200",
+		"ip -6 route show table 200",
+		// xfrm
```

Comment:
Not sure that this comment is all that helpful ;)

Reply:
Perhaps not overly helpful but it does match styling from above with `// iptables ` and `// Host and misc`. Seemed odd to pretend these were related to above section by not giving them their own title.

Reply:
Ah, must have been just out of view of the github code context view. Makes sense.